### DISCUSSION TITLE
Unit Test Project without Project Reference

### DISCUSSION THREAD

**Discussion by domtheluck** [stryker-mutator/stryker-net#1961] [Q&A]
Hi guys,

First of all, great tool :)!

My question is about project without project reference. In our case, we are not using project reference for some specific reasons so all our references are like that:

    <Reference Include="MyProject">
      <HintPath>$(SolutionDir)$(Configuration)\MyProject.dll</HintPath>
      <Private>false</Private>
    </Reference>

Looks like even by specifying the project to mutate in the command line, Stryker is not able to do anything.

This is a command line output:

**No project reference matched the given project filter 'MyProject.csproj'Choose one of the following references:**

Any idea to make it works?

Thanks

**Comment by rouke-broersma** [comment]
There is currently no way to make this work. You could create a feature request.
